DESCRIPTION

```diff
@@ -12,6 +12,11 @@ Authors@R:
            role = c("aut"),
            email = "contact@samabbott.co.uk",
            comment = c(ORCID = "0000-0001-8057-8037")),
+    person(given = "Emily",
+           family = "Tyszka",
+           email = "etsyzka@umass.edu",
+            role = c("ctb"),
```

Comment:
Did you mean to add her as an Author?

Reply:
Because they asked to hold off until they got permission. I can change to author.